app.logorrr.LogoRRR.yml

```diff
@@ -0,0 +1,36 @@
+app-id: app.logorrr.LogoRRR
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: LogoRRR
+modules:
+  - name: LogoRRR
+    only-arches: [aarch64]
```

Comment:
so what happen on the x86_64 flatpak?

Reply:
I could provide an archive containing the binaries for x86_64 architecture. 

But I'm again not sure how to provide a working .yml file which addresses both aarch64 and x86_64 correctly. i would assume i just create two identical module sections? 

Something like:

```
modules:
  - name: LogoRRR_aarch64
    only-arches: [aarch64]
    buildsystem: simple
    build-commands:
    ....
  - name: LogoRRR_x86_64
    only-arches: [x86_64]
    buildsystem: simple
    build-commands:
    ....
```

(the module build commands would look exactly the same)

I'm sure in hindsight this is will be trivial, but i couldn't find information for this.

Reply:
> I could provide an archive containing the binaries for x86_64 architecture.

Why is it not being built from source? Is it proprietary?

x86_64 should be provided.

Reply:
> i would assume i just create two identical module sections?

No, just the sources https://docs.flatpak.org/en/latest/module-sources.html#:~:text=skip%2Darches%20and%20dest.-,only%2Darches%20and%20skip%2Darches,-are%20used%20to

Reply:
ok i have to learn more about flathub and its approach. would it be possible to just go with the provided binaries for aarch64 until i figure out how to build the app from source in the sandboxed environment?

Reply:
We can wait.

Reply:
i've added a x64 build.